Cargo.toml

```diff
@@ -12,5 +12,8 @@ version = "0.9.1"
 [dependencies]
 dav1d-sys = { version = "0.7.0", path = "dav1d-sys" }
 
+[features]
+v1_1 = ["dav1d-sys/v1_1"]
```

Comment:
The CI is of course failing now because it runs with `--all-features`. So we'd need to pull in latest dav1d from git, which unfortunately still has the version in the pkg-config file at 1.0.0

Reply:
Time to fix that in dav1d maybe, even if might require some thinking regarding how to signal it is a quasi-versionless prerelease.

Reply:
That's not something we can fix on our side though. Do you want to bring that up in a dav1d issue that release management there needs some improvements?

Reply:
I started asking on IRC.